shell/app/locales/zh.json

```diff
@@ -2128,6 +2128,7 @@
     "earlier": "更早",
     "edit a endpoint": "编辑流量入口",
     "emergency": "紧急",
+    "edit monitoring": "编辑监控",
```

Comment:
where did u refer this translation?

Reply:
The code has just been updated. The conflict resolution side has overwritten the previous modifications